### ISSUE TITLE
How to load pretrain mode when training mode use my own dataset?

### ISSUE THREAD

**Issue by boyliwensheng** [TropComplique/FaceBoxes-tensorflow#15]
I what to use my dataset to train the mode .In order to shorter the time please tell me how to load pretrain mode,thanks!
